app/scripts/modules/core/src/domain/IManagedEntity.ts

```diff
@@ -34,3 +34,56 @@ export interface IManagedResource {
   managedResourceSummary?: IManagedResourceSummary;
   isManaged?: boolean;
 }
+
+export enum ManagedResourceEventType {
+  ResourceCreated = 'ResourceCreated',
```

Comment:
We have 2 more resource event types that I think we should include here - `ResourceTaskFailed` and `ResourceTaskSucceeded` - which actually indicates the orca task status of the resource.

Reply:
I originally wanted to include these, but none of them appear to have any sort of data attached to them about _which specific task they are in reference to_. I found it a bit confusing / weird to look at because there would be a `ResourceActuationLaunched` event with a specific task, then a bunch of other events (e.g. `ResourceDeltaResolved`, `ResourceValid`), then a `ResourceTaskFailed` or `ResourceTaskSucceeded` without any task info. Made it tough to follow exactly what was happening.

If we can get those events to include a `tasks` list the same way `ResourceActuationLaunched` has, I'll make sure to surface them here.

Reply:
Following our conversation, I doubled check and we currently do not log the task name. Adding an example payload for the `ResourceTaskFailed` event : `{"type":"ResourceTaskFailed","apiVersion":"titus.spinnaker.netflix.com/v1","kind":"cluster","id":"titus:cluster:titustestvpc:gyardeniapp-env4","application":"gyardeniapp","reason":"Too many retries.  10 attempts have been made to bake http://bakery.test.netflix.net:7001/api/v1/us-east-1/bake/b-4065khb8bt1frwg1jxjkw5b1y4 ... its probably not going to happen.","timestamp":"2020-02-11T23:27:02.989Z"}`. I'll be happy to add the task name if you think it's important info!

Reply:
Ah ok got it — thanks for checking! I'd love it if we could add the exact same `tasks` list that's [present on `ResourceActuationLaunched`](https://github.com/spinnaker/keel/blob/master/keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt#L200) to both `ResourceTaskFailed` _and_ `ResourceTaskSucceeded`. 

I made us a GitHub issue so we can keep track: https://github.com/spinnaker/keel/issues/829